### DISCUSSION TITLE
[web issue] players with existing player id number as name making profile pages impossible to see

### DISCUSSION THREAD

**Discussion by Ronikusu** [ppy/osu#23591] [Q&A] [answered]
### **issue**
if you have your name as a number(ex: 141), it makes the user page with that player id (ex: mc_XD) almost impossible to see. It redirects you to the other players page, in this case 141.

pictures

![image](https://github.com/ppy/osu/assets/124932615/a2b3ecfb-4211-404f-badb-030f85a12551)

### **the question**

is this a known issue?

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu-web/issues/9532#issuecomment-1555889832
